kibana-details/impersonation.md

```diff
@@ -0,0 +1,137 @@
+# Impersonation 
+
+After describing what [the impersonation is](../kibana.md#impersonate), it's high time to see how does ROR support it and who and when could be interested in using this feature. Let's start with the latter one.
+
+## Use cases
+
+The impersonation feature is prepared for a ROR admin. It's not rather useful for users (at least not directly). We can point out two the most obvious use cases when the admin could take advantage of the feature:
+
+#### Debugging users' problems:
+
+Let's imagine that some user has a problem with its ROR configuration (eg. the user doesn't has an access to some feature that was blocked at ROR's level by you, the admin). They are not able to clearly describe what the issue is (sounds familiar?). For the admin, it'd be better if they can see what the user sees. Thanks to impersonation feature, an admin is allowed to impersonate the user and feel and see the same the user sees. 
+
+#### Configuring a new user:
+
+When an admin configures a new user in ROR settings, they face two problems:
+
+1. `Won't the updated configuration break the production cluster?`
+1. `How do I know that the new user is correctly configured? Do they have correct permissions?`
+
+Both of the problems can be solved using the ROR's impersonation. Thanks to the fact that the impersonation operates on different than production (main) settings, the admin can alter it without worries that their actions will break something and users won't be able to do their job. 
```

Comment:
Thanks to the fact that the impersonation feature always uses its own "test settings", that is completely independent from the main production settings